### PR TITLE
update filter

### PR DESCRIPTION
**Pull request by ieguinoa**
Added the inputs for the fltering of matched peptides in the apex case + Corrected a typo.
This is the version currently available at usegalaxy.be

### CODE REVIEW

**Review by Maux82 — COMMENTED** [review]
in case of mbr : I have peptide_summary output that  it should not be.(indeed it fails it does not end the task).  I checked in the code and I dont figure out where this happen.

**Comment by Maux82** [comment]
@ieguinoa your last  commit  is it a  bit extreme ? the file is empty.

**Comment by ieguinoa** [comment]
@Maux82 sorry! galaxy screwed my file, there it is, I updated the reference to the ptm file, this should be referred as task.match_filter.ptm_file 
but the input for it should be checked:
-the type is defined as tabular, is this correct?
-it is defined as optional but in case there is no input defined it defaults to ptm_setting_ps.json which is not there///

File: tools/moFF/moff.xml [hunk 1]
```diff
@@ -0,0 +1,400 @@
...
+                --rt_peak_win $task.rt_peak_win
+	        --loc_out ./out
+                    #if ($task.match_filter.match_filter_selection):
+                        --match_filter
+                        --sample_size $task.match_filter.sample_size
+                        --quantile_thr_filtering $task.match_filter.quantile_thr_filtering
+                        #if ($task.match_filter.ptm_file):
+                            --ptm_file '$task.match_filter.ptm_file'
+                        #else:
+                            --ptm_file ./ptm_setting_ps.json
```
Comment (Maux82) [inline tools/moFF/moff.xml]: here , we should specify the path for the default ptm file. Actually the file it is in the moFF repository, I dont  know which the path of the file when it is executed inside moFF conda enviroment.  Probably the path used here is broken for sure.

File: tools/moFF/moff.xml [hunk 2]
```diff
@@ -0,0 +1,400 @@
...
+                   --peptide_summary
+               #end if
+               #if ($task.match_filter.match_filter_selection):
+                    --match_filter
+                    --sample_size $task.sample_size
+                    --quantile_thr_filtering $task.quantile_thr_filtering
+                    #if ($ptm_file):
+                        --ptm_file '$task.ptm_file'
+                    #else:
+                        --ptm_file ./ptm_setting_ps.json
```
Comment (Maux82) [inline tools/moFF/moff.xml]: Same  consideration also here.

**Review by Maux82 — COMMENTED** [review]
but the input for it should be checked:
> -the type is defined as tabular, is this correct? 
tecnically it is a json file, I dont know how Galaxy handles json file. 
> -it is defined as optional but in case there is no input defined it defaults to ptm_setting_ps.json which is not there///
Checks the comment in your commited version. We have to figure out it what it the  path of the default .json file in the conda env.

**Comment by ieguinoa** [comment]
ok, just added the default file as part of the tool dir.

**Review by Maux82 — APPROVED** [review]

**Comment by ieguinoa** [comment]
@Maux82 but should I change the input type to json? or what filetypes can the --ptm option take?

**Comment by Maux82** [comment]
> @Maux82 but should I change the input type to json? or what filetypes can the --ptm option take?

it takes only a json. if galaxy has  json file type , it is better to use it.

**Pull request merged**